### PR TITLE
Price book updates

### PR DESCRIPTION
**Pull request by Ation**
- definitions required for PB request 
- replace system_clock in adapter bus with steady_clock to protect timers from system clock changes.

### CODE REVIEW

File: BlocksettleNetworkingLib/Message/Envelope.h [hunk 1]
```diff
@@ -19,7 +19,12 @@
 namespace bs {
    namespace message {
       using UserValue = int;
-      using TimeStamp = std::chrono::time_point<std::chrono::system_clock>;
+      using bus_clock = std::chrono::steady_clock;
```
Comment (sergey-chernikov) [inline BlocksettleNetworkingLib/Message/Envelope.h:22]: Nice change, but it should go to terminal_arch_refactor branch first (then merged to futures_dev). Please consider splitting this PR.

File: Blocksettle_proto/Communication/bs_server.proto [hunk 2]
```diff
@@ -1578,6 +1583,7 @@ message PriceStreamManaging
       {
          string   owner_entity_name = 1;
          double   price             = 2;
+         int64    timestamp         = 3;
```
Comment (sergey-chernikov) [inline Blocksettle_proto/Communication/bs_server.proto:1586]: Why int64 and not uint64? Isn't timestamp always an absolute unix time?

File: BlocksettleNetworkingLib/Message/Envelope.h (hunk 1 above)
Comment (Ation) [reply BlocksettleNetworkingLib/Message/Envelope.h:22]: You could cherry pick 2 commits. They are isolated from other changes. Same in PB PR, this change is an isolated adapter.

File: Blocksettle_proto/Communication/bs_server.proto (hunk 2 above)
Comment (Ation) [reply Blocksettle_proto/Communication/bs_server.proto]: fixed

**Review by sergey-chernikov — APPROVED** [review]

**Pull request merged**